### PR TITLE
add a broadcast command for staff

### PR DESCRIPTION
**Pull request by StudentAlleg**
Implements #153

### CODE REVIEW

File: core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/BroadcastCommand.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        }
+
+        Component message = Component.empty().append(client.getRank().getPlayerNameMouseOver(player.getName()).decorate(TextDecoration.BOLD))
+                        .append(UtilMessage.deserialize(" <red>%s", String.join(" ", args)));
+
+        Sound notification = Sound.sound(Key.key(Key.MINECRAFT_NAMESPACE, "block.note_block.bell"), Sound.Source.NEUTRAL, 1.0f, 1.0f);
+        for (Player playerToSend : Bukkit.getOnlinePlayers()) {
+            UtilMessage.message(playerToSend, message);
+            playerToSend.playSound(notification, Sound.Emitter.self());
+        }
```
Comment (Mykindos) [inline core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/BroadcastCommand.java:58]: UtilSound should be able to do this without needing Key.key, just Sound.BLOCK_NOTE_BLOCK_BELL

File: core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/BroadcastCommand.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    @Override
+    public Rank getRequiredRank() {
+        return Rank.HELPER;
```
Comment (Mykindos) [inline core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/BroadcastCommand.java:63]: probably an admin command, can remove this method completely (i think)

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/BroadcastCommand.java (hunk 2 above)
Comment (StudentAlleg) [reply core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/BroadcastCommand.java]: implemented

File: core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/BroadcastCommand.java (hunk 1 above)
Comment (StudentAlleg) [reply core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/BroadcastCommand.java:48]: Implemented

**Review by Mykindos — APPROVED** [review]

**Pull request merged**